mmv1/third_party/tgc/convert.go

```diff
@@ -172,13 +173,20 @@ func (c *Converter) addDelete(rc *tfjson.ResourceChange) error {
 		}
 
 		for _, converted := range convertedItems {
-
 			key := converted.Type + converted.Name
 			var existingConverterAsset *resources.Asset
 			if existing, exists := c.assets[key]; exists {
 				existingConverterAsset = &existing.converterAsset
 			} else if !c.offline {
-				asset, err := converter.FetchFullResource(rd, c.cfg)
+				var asset cai.Asset
+				var err error
+				err = transport_tpg.Retry(transport_tpg.RetryOptions{
```

Comment:
Rather than having the retry handling here, it should be inside the converter.FetchFullResource function so that the error retry / abort predicates can be included - similar to https://github.com/GoogleCloudPlatform/magic-modules/blob/a8ab34eb06ae386e8394ba6a690c33d84a235aa6/mmv1/templates/terraform/resource.erb#L271 (except that calls SendRequestOptions, which is a wrapper around Retry. We can still use Retry directly, we just need to make sure we can supply it with the retry / abort predicates, which are per-resource, so are not knowable here.)

Reply:
https://github.com/hashicorp/terraform-provider-google-beta/blob/main/google-beta/transport/transport.go#L55
I think this PR can be abandoned. 
As long as using transport_tpg.SendRequest, it has retry logic baked in. 
I checked all the FetchFullResource calls, and they are all eventually using SendRequest.

Reply:
Makes sense! SGTM.